docs/Macbook-pro-setup.md

```diff
@@ -0,0 +1,231 @@
+# MacBook Pro Set Up 
+
+This document contains instructions for setting up a MacBook Pro in B.C. Government, tailored for data science workflows used by the Data Science Partnerships team.  The document is organized with least complex steps at the top (i.e. point and click downloads) leading to tools that require more advanced setup at the bottom. Some of the setup instructions assume a working knowledge of Unix/Linux. 
```

Comment:
The second sentence may need revision as doc has been split.

Reply:
```suggestion
This document contains instructions for setting up a MacBook Pro in B.C. Government, tailored for data science workflows used by the Data Science Partnerships team.  The document is organized with least complex steps at the top (i.e. point and click downloads) leading to tools that require more advanced setup at the bottom. There is an [additional document](Macbook-advanced-geo.md) that contains some lesser-used advanced setup instructions as well as tips for setting up a geospatial data science environment. Some of the setup instructions assume a working knowledge of Unix/Linux command-line environments. 
```